docs/changehistory/NextVersion.md

```diff
@@ -26,3 +26,10 @@ Enums in TypeScript have some shortcomings, one of which resulted in a bug that
 ## Deprecations in @itwin/core-react package
 
 Using the sprite loader for SVG icons is deprecated. This includes [SvgSprite]($core-react) and the methods getSvgIconSpec() and getSvgIconSource() methods on [IconSpecUtilities]($appui-abstract). The sprite loader has been replaced with a web component [IconWebComponent]($core-react) used by [Icon]($core-react) to load SVGs onto icons.
+
+### Non-deferring transformer implementation
```

Comment:
I think in reality the defer-based implementation also could call onTransformElement each time it retried exporting a deferred elements so i will confirm that and if so remove this section from NextVersion.md

Reply:
I confirmed it also runs once per element deferral. Therefore removing this since it isn't a change.

Reply:
removed this next version detail.